new.go

```diff
@@ -149,7 +149,7 @@ func resolveImage(ctx context.Context, systemContext *types.SystemContext, store
 			pulledImg, pulledReference, err := pullAndFindImage(ctx, store, image, options, systemContext)
 			if err != nil {
 				logrus.Debugf("unable to pull and read image %q: %v", image, err)
-				continue
+				return nil, nil, err
```

Comment:
Darn, got the tests to run but found an issue.  We need to do the continue here as it trys to pull from the local registry at one point even if it's not used, that's done before it looks out at quay.io or whatever is in registries.conf.  Back to the drawing board.